### ISSUE TITLE
Cannot connect to Rokoko Livelink source in a packaged build.

### ISSUE THREAD

**Issue by DavidWLejeune** [Rokoko/rokoko-studio-live-unreal-engine#33]
When running a packaged build the Livelink connection will fail with the following errors:

LogStreaming: Error: Couldn't find file for package /Script/SmartsuitEditor requested by async loading code. NameToLoad: /Script/SmartsuitEditor
LogLiveLink: Warning: Create Source Failure: The factory is not defined.
LogLiveLink: Warning: Create Subject Failure: The source doesn't exist.
LogLiveLink: Warning: Create Subject Failure: The source doesn't exist.
LogLiveLink: Error: Could not apply [LiveLink preset uasset name]

**Comment by shazamk** [comment]
I'm running into this issue as well. According to the Smartsuit.uplugin file, the SmartsuitEditor folder is marked as "UncookedOnly" which I'm assuming means its not included in runtime builds?  Unfortunately there are files in that folder (I think VirtualProductionSourceFactory) which are needed to create the Rokoko Live Link. Based on this it seems like Rokoko Live wasn't designed to be used in builds? Any updates from the devs?

**Comment by shazamk** [comment]
I managed to resolve this issue and get Rokoko Livelink to work in an Unreal build. I had to move the VirtualProductionSourceFactory
and VirtualProductionSourceEditor files from the SmartsuitEditor folder to the Smartsuit folder then rebuild the project sln.

**Comment by nobeknia** [comment]
Hi! I tried to do what @shazamk suggest but it doesn't work to me, I don't know If I'm missing something, in the editor the gloves are working, but when I make the build i quit the editor and run the build and seems that is not getting any realtime movement. What I noticed is that everytime I enter in the editor, I have to add manually the rokoko studio source, this could be the problem? I'm a bit lost...
